src/dashboard/get_chart_data/templates/get_chart_data.sql.jinja

```diff
@@ -0,0 +1,65 @@
+{%- import 'syntax.sql.jinja' as syntax -%}
+{%- import 'filter_numeric.sql.jinja' as numeric -%}
+{%-macro select_data(cumulus__none) -%}
+    SELECT 
+        {%- if stratifier_column %}
+        "{{ stratifier_column }}",
+        {%- endif %}
+        "{{ data_column }}"
+        {%- if count_columns %},{%- endif  %}
+        {%- for column in count_columns  %}
+        SUM("{{ column }}") AS {{ column }}{{ syntax.comma_delineate(loop) }}
+        {%- endfor%}
+    FROM "{{ schema }}"."{{ data_package_id }}"
+    WHERE
+        {%- if coalesce_columns  %} COALESCE (
+        {%- for column in coalesce_columns  %}
+            cast("{{ column }}" AS VARCHAR){{ syntax.comma_delineate(loop) }} 
+        {%- endfor%}
+        ) IS NOT NULL
+        AND{%- endif%} "{{ data_column }}" IS NOT NULL
+        {%- if not cumulus__none %}
+        AND {{ data_column }} !='cumulus__none'
+        {%- if filter_str %}
+        AND {{ filter_str }}
+        {%- endif  %}
+        {%- else %}
+        AND {{ data_column }} ='cumulus__none'
+        {%- endif  %}
+        {%- if stratifier_column %}
+        AND "{{ stratifier_column }}" IS NOT NULL
+        {%- endif  %}
+    GROUP BY
+        {%- if stratifier_column %}
+        "{{ stratifier_column }}", "{{ data_column }}"
+        {%- else %}
+        "{{ data_column }}"
+        {%- endif %}
```

Comment:
I will point out here that because you don't care about the prettiness of the resulting SQL (right?), you can instead try to make the jinja code more readable / maintainable.

Things like reducing the duplication here:
```
        {%- if stratifier_column %}
        "{{ stratifier_column }}",
        {%- endif %}
        "{{ data_column }}"
```

And/or proper indenting throughout for the if/else blocks, plus whatever else. The fact that the jinja in Library is usually hard to read is _partly_ our fault.

Reply:
> yeah i think we can ~~make~~ **ditch** some compromises over here that we cant/wont in the library.

Fixed that for ya :smile: